admin/src/scenes/centersV2/list.js

```diff
@@ -294,6 +294,18 @@ const ListCenter = ({ firstSession }) => {
   ];
   if (user.role === ROLES.ADMIN) filterArray.push({ title: "Code", name: "code2022", missingLabel: "Non renseignée" });
 
+  const OrderCohort = (array) => {
+    for (const session of array) {
+      if (Object.prototype.hasOwnProperty.call(COHESION_STAY_START, session.key)) {
+        session.date = COHESION_STAY_START[session.key];
+      } else {
+        session.date = new Date(2000, 0, 1);
+      }
+    }
+    array.sort((a, b) => b.date - a.date);
+    return array;
+  };
```

Comment:
Juste j'ai pas compris pourquoi ça s'appelle `session` et pas `cohort`.

En fait ça aurait pu être `for (const cohort of cohorts) {`